learn/index.html

```diff
@@ -154,6 +154,14 @@
 
       <ul class="cards">
         <li>
+          <h3>Lunch &amp; Learn</h3>
+          <p>"Better know a language: PureScript" by <a href="http://taylor.fausak.me">Taylor Fausak</a></p>
```

Comment:
Actually, if the title went in the `<h3>`, it wouldn't really need quotes, right? 

``` html
<h3>Better know a language: PureScript</h3>
<p>by …</p>
```